portal-impl/src/com/liferay/portal/jsonwebservice/JSONWebServiceActionImpl.java

```diff
@@ -682,9 +682,13 @@ public Date convert(Object object) {
 
 		private static final DateTimeFormatter _dateTimeFormatter =
 			new DateTimeFormatterBuilder().parseCaseInsensitive(
-			).append(
+			).appendOptional(
 				DateTimeFormatter.ISO_LOCAL_DATE
 			).optionalStart(
+			).appendPattern(
+				"yyyy-M-d"
```

Comment:
@dantewang Just want to confirm is "yyyy-M-d" the only one missing?

Tina.

Reply:
@tinatian yyyy-MM-dd + yyyy-M-d should cover what the old jodd supports for dates.

Reply:
@dantewang But here only "yyyy-M-d" is set, what about the other one?

Reply:
@tinatian `ISO_LOCAL_DATE` is `yyyy-MM-dd`.